ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -57,7 +57,7 @@ const string Op_ArmMoveJoint           = "ArmMoveJoint";
 const string Op_ArmMoveJoints          = "ArmMoveJoints";
 const string Op_DigCircular            = "DigCircular";
 const string Op_DigLinear              = "DigLinear";
-const string Op_Deliver                = "Deliver";
+const string Op_TaskDeliverSample      = "TaskDeliverSample";
```

Comment:
This change (and its many dependencies) is optional.  The constants can be convenient short names for the actions -- they don't have to match the action names.   I realize that some already do.